cloudinary/apiCaller.js

```diff
@@ -45,7 +45,7 @@ const sendToCloudinery = (imagesArray, batchSize, dpr, metaData, cb, rollBarMsg)
     let eager = [];
     if (transformations) {
       eager = transformations.map((trans) => {
-        return Object.assign({}, trans, {width: image.width, height: image.height, dpr: dpr || 1});
+        return Object.assign({}, trans, {width: image.width, height: image.height, dpr: dpr || 1, quality: 'auto:' + quality});
```

Comment:
Theoretically, `q_auto` and `q_auto:good` have slightly different meanings and could have different results, thus I would keep the option to not pass anything in here and leave it as `quality: auto`.

Reply:
changed it